client/src/components/Sidebar/Sidebar.jsx

```diff
@@ -78,7 +81,7 @@ const Sidebar = ({ disabled, showUIPatterns, loading }) => {
   const addItems = [
     {
       className: styles['feedback-nav-item'],
-      children: <FeedbackButton />,
+      children: !hideFeeback && <FeedbackButton />,
     },
```

Comment:
Should the`!hideFeedback` determine inclusion of the entire object in `addItems`? Otherwise an object is still being passed with a className but null children

Reply:
Good point, I had gone back and forth on that, but I just went ahead and added a `hidden` attribute to the object. This also brings it more in line with the rest of the sidebar items affected by workbench settings.